app/src/keymap.c

```diff
@@ -134,24 +145,50 @@ int zmk_keymap_apply_position_state(int layer, uint32_t position, bool pressed,
     behavior = device_get_binding(binding->behavior_dev);
 
     if (!behavior) {
-        LOG_DBG("No behavior assigned to %d on layer %d", position, layer);
+        LOG_WRN("No behavior assigned to %d on layer %d", position, layer);
```

Comment:
Did this fall out of testing?

Reply:
Crept in, yeah. Was easier to spot in log output, and felt like a better level.